knative-kubecon/build/010-build.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: Build
+metadata:
+  name: oc-build-1
+spec:
+  source:
+    git:
+      url: https://github.com/vdemeester/os-sample-python
```

Comment:
the idea is to hook that code also to eventing, using a `ksvc`, right @vdemeester @markusthoemmes  ?

Reply:
@matzew in the end, I think so yeah :angel: this PR "just" import the raw build in the repo, to be mangled, updated to the final demo script/workflow/data :angel:

Reply:
@matzew correct, we're creating a service already in the 020-serving.yaml to be connected to eventing.